libs/s25main/notifications/NotificationManager.h

```diff
@@ -35,22 +35,22 @@ class NotificationManager
     class CallbackUnregistrar;
 
 public:
-    inline NotificationManager();
-    inline ~NotificationManager();
+    NotificationManager();
+    ~NotificationManager();
 
     /// Subscribe to a specific notification.
     /// Unsubscribes when the subscription has no references left
     template<class T_Note>
-    inline Subscribtion subscribe(std::function<void(T_Note)> callback);
+    Subscribtion subscribe(std::function<void(T_Note)> callback);
```

Comment:
can we fix that typo too? `Subscri**p**tion`

Reply:
Good point: https://github.com/Return-To-The-Roots/s25client/pull/1227/commits/5ba8395eed50885e5c2ef22f3d2d1a0c107eacc8